Formula/r/rna-star.rb

```diff
@@ -0,0 +1,40 @@
+class RnaStar < Formula
+  desc "RNA-seq aligner"
+  homepage "https://github.com/alexdobin/STAR"
+  url "https://github.com/alexdobin/STAR/archive/refs/tags/2.7.11b.tar.gz"
+  version "2.7.11b"
+  sha256 "3f65305e4112bd154c7e22b333dcdaafc681f4a895048fa30fa7ae56cac408e7"
+  license all_of: ["MIT", "BSD-2-Clause"]
+
+  uses_from_macos "vim" => :build # needed for xxd
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  def install
+    cd "source" do
+      args = ["CXXFLAGS_SIMD="]
+      if OS.linux?
+        args << "STAR"
+      else
+        args += [
+          "LDFLAGS_Mac_static=-pthread -lz htslib/libhts.a", # remove `-static-libgcc`
+          "STARforMacStatic",
+        ]
+      end
+      system "make", *args
+      bin.install "STAR"
+    end
```

Comment:
As you confirmed, if it is not different then it's better to make it shorter. This does not install libraries and so I think we don't need to consider the static dynamic one.
```suggestion
    args = ["CXXFLAGS_SIMD="]
    args << "CXXFLAGSextra=-D'COMPILE_FOR_MAC'" if OS.mac?

    cd "source" do
      system "make", "STAR", *args
      bin.install "STAR"
    end
```